### ISSUE TITLE
ALiVE and RHS: wierd message

### ISSUE THREAD

**Issue by apargo** [ALiVEOS/ALiVE.OS#320]
Hi! 

I'm using ALiVE with RHS and I get a couple of strange messages in the .rpt file:

Warning Side: EAST Faction: rhs_faction_msv could not find a SpecOps group
Warning Side: EAST Faction: rhs_faction_msv could not find a Air group
Warning Side: WEST Faction: rhs_faction_usarmy_wd could not find a Air group
Warning Side: WEST Faction: rhs_faction_usarmy_wd could not find a SpecOps group

I'm new with this amazing mod so I'm pretty lost here. I'd appreciate someone who might bring some light.

Thanx a lot.

**Comment by highhead** [comment]
Hi comrade!

Thanks for your feedback! This basically means, that ALiVE tried to create f.e. a Spec-Ops group (snipers, divers etc.) for rhs_faction_msv (not sure whats the real name of this faction in editor) but there are no spec-ops groups defined for that faction. If your mission is running fine, and there are enough other groups present for that faction, you could just ignore it.

If there are no groups created at all or very few, you may want to use another RHS faction or use the "ORBAT creator" to create the given missing groups for that faction manually.

Please reply, if this doesn't answer your questions and you still feel stuck! Thanks!
Highhead

**Comment by highhead** [comment]
ps: please zip the mission instead of pboing, saves me some time depboing

**Comment by apargo** [comment]
Thank for your reply.

If I Choose the default factions (NATO and CSAT) obviously everything works fine. As for the rest:

rhs_faction_usarmy_wd: It has helicopters so I don't know why that message shows up. I'll try to use ORBAT.

rhs_faction_msv: It doesn't have helicopters so I should used also a second faction for air units (rhs_faction_vdv for instance. rhs_faction_vvs -VVS = Russian Air Force- is not supported according to the documentation). 

That brings me to the following: Can I use two factions at the same time for one side?. If I try to add two RHS factions in the corresponding fields of the OPF faction (override faction, force faction and auto-task faction fields)  separated by a coma (rhs_faction_msv ,rhs_faction_vdv) in the modules:

Military AI Commander
Military Placement (civil and military obj)
Military Close Quarters Battle
Player Command/Control

I get the following error:

There are are no groups for OPCOM faction(s) rhs_faction_msv! Please ensure you have configured a Mil Placement or Mil Placement (Civ Obj) module for this faction (or faction units are synced to Virtual AI module). If so, please check groups are correctly configured for this faction.
There are are no groups for OPCOM faction(s) rhs_faction_vdv! Please ensure you have configured a Mil Placement or Mil Placement (Civ Obj) module for this faction (or faction units are synced to Virtual AI module). If so, please check groups are correctly configured for this faction.
There are are no groups for OPCOM faction(s) ["rhs_faction_msv","rhs_faction_vdv"]! Please check you chose the correct faction(s), and that factions have groups defined in the ArmA 3 default categories infantry, motorized, mechanized, armored, air, sea!

Maybe I'm doing something wrong or it doesn't support overriding two factions at the same time (?). Spec Op Groups are not a big deal so I can live without them but air units is a totally different issue. Anyway I though I saw an enemy air unit yesterday although it killed me before I could take a deeper look at it.

I'll keep researching. Thanks again.

**Comment by HeroesandvillainsOS** [comment]
rhs_faction_msv groupings are messed up since the last batch of RHS updates. As far as I can tell, the only Russian faction that still seems to work ok with ALiVE right now rhs_faction_vdv, though be warned they aren't sorted by camo which kind of sucks TBH.

Using two working factions for the same side is perfectly fine. Both need to be defined in the Military AI Commander module (separated by a comma with no spaces), but you can only use one faction in each MIL Obj/CIV Obj module.

Though note, I think you can only call logistics support for the faction you are playing as. For example, if you're playing as a soldier from rhs_faction_vdv, I don't think you'll be able to use logistics (reinforcements, etc) for rhs_faction_msv.

At least I'm pretty sure that's how it works. There are probably other limitations like this too. I don't tend to use multiple factions for the same side too often, so I'm not too sure.

Long story short, I think all of your issues are related to RHS. Probably even the wd US Army faction and helicopters too. Not sure, I usually only use desert Marines and Army, but don't let helicopters spawn. But regardless, the working US factions seem to be working better than the Russian ones. MSV I think is pretty messed up right now! Probably best if you ORBAT a faction yourself or drop the Russians, unless you fancy VSV and their assortment of camo patterns.

**Comment by friznit** [comment]
RHS mappings in dev for 1.3.4
